Added Hungarian locale for Portal

eszterik opened the pull request:
Hello! I've rewritten the Hungarian locale because I found the current version quite problematic. As a native Hungarian speaker, I noticed that some texts are missing, while others contain misspellings and sound overly robotic. Native Hungarian speakers don't typically use the expressions or sentence structures present in the original version. 

Additionally, I changed the polite second formal pronouns to an informal and friendly tone. (In Hungarian, this is similar to the distinction between the formal and informal "Sie/sie" in German.) For newsletters and new media formats, we generally don't use the formal style; it comes across as somewhat outdated and doesn't align with the Ghost.org style either.

I hope you like my new version.

As for my qualifications, I am a native Hungarian speaker, journalist, and copywriter. I am open to discussing my changes further. All the best, and I look forward to hearing from you!

eszterik committed:
Added Hungarian locale for Portal

cathysarisky commented on ghost/i18n/locales/hu v2/portal.json:
```diff
@@ -0,0 +1,209 @@
+    "Your account": "Fiókod",
+    "Your email has failed to resubscribe, please try again": "Az e-mail-címedet nem sikerült újra regisztrálni. Kérjük, próbáld újra",
+    "Your input helps shape what gets published.": "A visszajelzésed segít abban, hogy miről írjunk",
+    "Your subscription will expire on {{expiryDate}}": "Az előfizetésed lejár ekkor: {{expiryDate}}",
+    "Your subscription will renew on {{renewalDate}}": "Az előfizetésed megújul ekkor: {{expiryDate}}",
+    "Your subscription will start on {{subscriptionStart}}": "Az előfizetésed ekkor indul: {{expiryDate}}"
```
Code Rabbit is correct - this is a must-fix item.

eszterik replied:
Thank you, Cathy for confirming.

cathysarisky reviewed (changes requested):
Hey @eszterik, thanks for these contributions!  

Please review the comments by CodeRabbit - it's correct in flagging your errors in variable names.  Those must be corrected.

Additionally, these files need to be submitted as edits to /locales/hu/<filename>.json, not as a separate directory.

cathysarisky commented on ghost/i18n/locales/hu v2/portal.json:
```diff
@@ -0,0 +1,209 @@
+    "Expires {{expiryDate}}": "Lejár: {{expiryDate}}",
+    "Failed to cancel subscription, please try again": "Nem sikerült törölni az előfizetést. Kérjük, próbáld újra",
+    "Failed to log in, please try again": "Nem sikerült bejelentkezni. Kérjük, próbáld újra",
+    "Failed to log out, please try again": "Nem sikerült kijelentkezni. Kérjük, róbáld újra",
```
Please recheck - is it próbáld ?

eszterik replied:
Yes, it is. Do I fix in the file?

cathysarisky replied:
Yes, please. You should be able to make the change and commit, and it'll show up here on the PR.

cathysarisky replied:
Hey @coderabbitai , please learn that if there is a typo or spelling error within ghost/i18n/locales/*.json , this is an important problem, not a nitpick.  Please flag these.

cathysarisky commented on ghost/i18n/locales/hu v2/portal.json:
```diff
@@ -0,0 +1,209 @@
+    "{{amount}} days free": "{{amount}} napig ingyen",
+    "{{amount}} off": "{{amount}} kedvezmény",
+    "{{amount}} off for first {{number}} months.": "{{amount}} kedvezmény az első {{number}} hónapig.",
+    "{{amount}} off for first {{period}}.": "{{amount}} kedvezmény eddig: {{period}}",
```
ChatGPT flags this as unclear and suggests that {{amount}} kedvezmény az első {{period}} időszakra is clearer.

I'm not a native speaker, so will absolutely defer to your judgement, but it may be worth another look.  What's going to to in for {{period}} is something like "2 weeks" or "1 day"

eszterik replied:
Yes:  kedvezmény az első {{period}} időszakra - this is good! I should fix, right?

cathysarisky commented on ghost/i18n/locales/hu v2/ghost.json:
```diff
@@ -0,0 +1,34 @@
+    "Thank you for signing up to {{siteTitle}}!": "{{siteTitle}} — Köszönjük a regisztrációt!",
+    "Thank you for subscribing to {{siteTitle}}!": "{{siteTitle}} — Köszönjük a feliratkozást!",
+    "Thank you for subscribing to {{siteTitle}}.": "{{siteTitle}} — Köszönjük a feliratkozást!",
+    "Thank you for subscribing to {{siteTitle}}. Tap the link below to be automatically signed in:": "Köszönjük a feliratkozást! Kérjük, kattints az alábbi linkre az automatikus bejelentkezéshez:",
```
You need to use the {{siteTitle}} variable in the translations.

eszterik replied:
Okay!

cathysarisky commented on ghost/i18n/locales/hu v2/ghost.json:
```diff
@@ -0,0 +1,34 @@
+{
+    "All the best!": "Üdvözlettel,",
+    "Complete signup for {{siteTitle}}!": "{{siteTitle}} —\u00a0Erősítsd meg a feliratkozásodat a hírlevélre!",
```
Not all Ghost sites have a newsletter.  Some take sign-ups for web-only access.  This language needs adjustment.

cathysarisky reviewed (changes requested):
Hey @eszterik, thanks for these contributions!  

Please review the comments by CodeRabbit - it's correct in flagging your errors in variable names.  Those must be corrected.

Additionally, these files need to be submitted as edits to /locales/hu/<filename>.json, not as a separate directory.

A few other comments as well based on my AI-based review.  I'm not a Hungarian speaker, so feel free to override, but I've often found that many AI flags are worth considering.

eszterik committed:
Update ghost/i18n/locales/hu v2/newsletter.json

"You are receiving this because you are a <strong>%%{status}%% subscriber</strong> to {site}.": "Ezt azért kapod, mert <strong>%%{status}%% feliratkoztál</strong> ide: {site}",

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

eszterik commented on ghost/i18n/locales/hu v2/newsletter.json:
```diff
@@ -0,0 +1,24 @@
+    "Unsubscribe": "Leiratkozás",
+    "View in browser": "Megnyitás a böngészőben",
+    "You are receiving this because you are a <strong>%%{status}%% subscriber</strong> to {site}.": "Ezt azért kapod, mert <strong>%%{status}%% subscriber</strong> to {site}",
+    "Your free trial ends on {date}, at which time you will be charged the regular price. You can always cancel before then.": "Az ingyenes próbaidőszak után fiókod automatikusan átvált előfizetésesre. A próbaidőszak alatt bármikor lemondhatod az előfizetésedet.",
```
Should I fix it?

cathysarisky replied:
Yes. Translations should have the same information content.

cathysarisky replied:
Yes, you should be submitting an edited version of the /hu/ files, not making a new directory.  (A separate directory would mean users would have to type something like "hu v2" as their language, if that actually worked, which it wouldn't because it isn't on the locales list.)

eszterik replied:
Thank you, Cathy! I copied the hu2 files to hu. Do I delete the hu2? 
And do I have to send the refreshed "hu" to another review?

eszterik replied:
Hello Cathy, Could you check .hu locales? I copied the hu2 content to hu (files by files), but it seems not okay. See 'M'. I am sorry, I am not a programmer, and this is the first time when I use GitHub. Thank you.

cathysarisky commented on ghost/i18n/locales/hu v2/newsletter.json:
```diff
@@ -15,7 +15,7 @@
     "View in browser": "Megnyitás a böngészőben",
-    "You are receiving this because you are a <strong>%%{status}%% subscriber</strong> to {site}.": "Ezt azért kapod, mert <strong>%%{status}%% subscriber</strong> to {site}",
+    "You are receiving this because you are a <strong>%%{status}%% subscriber</strong> to {site}.": "Ezt azért kapod, mert <strong>%%{status}%% előfizető</strong> vagy itt: {site}",
     "Your free trial ends on {date}, at which time you will be charged the regular price. You can always cancel before then.": "Az ingyenes próbaidőszak után fiókod automatikusan átvált előfizetésesre. A próbaidőszak alatt bármikor lemondhatod az előfizetésedet.",
```
This problem remains, that the {date} section is missing.

cathysarisky commented:
Hi @eszterik , I am still seeing hu v2 for your directory name, when it should be hu.  You're right that you need to copy the changes to the hu directory, and delete the hu v2 directory.  

Probably the best way to do that is to go into your copy of the repository (https://github.com/eszterik/Ghost/tree/main) and press . (the period key), then open folders on the left side.  Right click on the hu folder and choose delete.  Then right click on hu v2 and rename it to hu.  Then commit your changes.  That'll update the PR.  You'll know you've done it right if you view the "files changed" section of the PR and see files in /hu/, with no sign of /hu v2/.

cathysarisky commented:
I'm not going to review this further as I see that @peterzimon has self assigned, and he's far better qualified to review the translations than I am.  (But you DO need to fix the folders.)

cathysarisky commented:
> > > `18-18`: **Consider standardizing placeholder style.**
> > > The translation uses two different placeholder styles: `%%{status}%%` and `{site}`. Consider standardizing to use curly braces consistently if possible.

@coderabbitai , this is a bad suggestion.  The keys in these json files should not be changed by translators.  The %%{status}%% placeholder is required for string replacement by Mailgun and is correct just as it is.  Keys should not be changed if a PR is just making corrections/additions to translation files in ghost/i18n/locales

cathysarisky commented:
Apologies to the humans for the extra comment spam.  Trying to teach our AI bunny how to be its best self. 😆

eszterik commented:
> Hi @eszterik , I am still seeing hu v2 for your directory name, when it should be hu. You're right that you need to copy the changes to the hu directory, and delete the hu v2 directory.
> 
> Probably the best way to do that is to go into your copy of the repository (https://github.com/eszterik/Ghost/tree/main) and press . (the period key), then open folders on the left side. Right click on the hu folder and choose delete. Then right click on hu v2 and rename it to hu. Then commit your changes. That'll update the PR. You'll know you've done it right if you view the "files changed" section of the PR and see files in /hu/, with no sign of /hu v2/.

@cathysarisky 
if I would like to delete this folder, it is said:
This action cannot be undone

Is it okay?
I am sorry, I would like to avoid any big mistakes.

cathysarisky commented:
You won’t hurt Ghost. The very worst that could happen is that you lose your work (so make a copy first in your local computer).

eszterik commented:
Okay, thanks! I did it.

cathysarisky commented:
> Okay, thanks! I did it.

I'm not seeing it. It still shows up as changes in hu v2.  Happy to review this and queue it up for merge, but this needs fixing first.  🙏

eszterik commented:
Hello @cathysarisky, I'm sorry, I am not sure what I should do at the moment. I followed your instructions, and it seemed okay. I attached a screenshot because I see only .hu folder. I really appreciate your support, thank you! 
Please let me know what to do now.
Best, Eszter
![Képernyőfotó 2025-02-11 - 11 36 25](https://github.com/user-attachments/assets/8e2c16f6-e071-4cfb-9cb8-ed3f44b7de64)

eszterik committed:
Added new Hungarian locale for Portal

eszterik committed:
Added new Hungarian locale for Portal

cathysarisky commented on ghost/i18n/locales/hu/portal.json:
```diff
@@ -146,64 +146,64 @@
+    "You're not receiving emails because you either marked a recent message as spam, or because messages could not be delivered to your provided email address.": "Nem kapsz e-maileket, mert vagy spamnek jelöltél egy tőlünk kapott e-mailt, vagy mert a megadott e-mail-cím nem tud üzeneteket fogadni.",
+    "You've successfully signed in.": "Sikeresen bejelentkeztél.",
+    "You've successfully subscribed to": "Sikeresen bejelentkeztél ide: ",
+    "Your account": "Fiókod",
+    "Your email has failed to resubscribe, please try again": "Az e-mail-címedet nem sikerült újra regisztrálni. Kérjük, próbáld újra",
+    "Your input helps shape what gets published.": "A visszajelzésed segít abban, hogy miről írjunk",
```
@eszterik this is a must-fix item. Thank you!

cathysarisky reviewed (changes requested):
You fixed the folder issue, hooray! :)

There are a couple comments from coderabbit that merit your review, and I'll also take a look over it on Monday.  (Feel free to fix the coderabbit-identified issues now -- no need to wait for me..)

cathysarisky commented on ghost/i18n/locales/hu/portal.json:
```diff
@@ -1,143 +1,143 @@
+    "Expires {{expiryDate}}": "Lejár: {{expiryDate}}",
+    "Failed to cancel subscription, please try again": "Nem sikerült törölni az előfizetést. Kérjük, próbáld újra",
+    "Failed to log in, please try again": "Nem sikerült bejelentkezni. Kérjük, próbáld újra",
+    "Failed to log out, please try again": "Nem sikerült kijelentkezni. Kérjük, próbáld újra",
+    "Failed to process checkout, please try again": "Nem sikerült bejelentkezni. Kérjük, próbáld újra",
```
This is the message that will display if something goes wrong with setting up for the user to pay for a plan.  Coderabbit seems to think you've got an error here, so please double check it. :)

cathysarisky commented on ghost/i18n/locales/hu/portal.json:
```diff
@@ -146,64 +146,64 @@
+    "Your email has failed to resubscribe, please try again": "Az e-mail-címedet nem sikerült újra regisztrálni. Kérjük, próbáld újra",
+    "Your input helps shape what gets published.": "A visszajelzésed segít abban, hogy miről írjunk",
+    "Your subscription will expire on {{expiryDate}}": "Az előfizetésed lejár ekkor: {{expiryDate}}",
+    "Your subscription will renew on {{renewalDate}}": "Az előfizetésed megújul ekkor: {{expiryDate}}",
```
@eszterik Please have a look at lines 207-208, where the variable name was changed.
@coderabbitai - nice job identifying this problem!

eszterik replied:
Hello @cathysarisky, I also checked the nitpick comments and fixed them where Rabbit was right:)
Kérlek - Kérjük - I ask you - We ask you (Please). Kérjük is not formal in this context, it is also used in informal way (actually, no other options).

cathysarisky commented:
There are some potentially-useful comments from coderabbit about tone, hidden within "nitpick" comments - have a look and see if it is right about anything, please.  (It may not be - I'm sure you speak better Hungarian than it does.)

cathysarisky commented:
Looks like you maybe forgot to actually commit your changes?  Back to the "merge" icon, which probably has a blue badge, and get those changes committed, please! :)

eszterik committed:
Added fixed Hungarian locale for Portal

eszterik committed:
Added fixed Hungarian locele for Portal

eszterik commented:
@cathysarisky Hello Cathy, do you have any updates on these changes? When will the updates go live on the websites? Thank you, Eszter

cathysarisky commented:
> @cathysarisky Hello Cathy, do you have any updates on these changes? When will the updates go live on the websites? Thank you, Eszter

Let me follow up on that.  I thought I'd put it on the list for merging, but either I missed it, or the member of the Ghost core team doing the merging missed it.  Will get it queued!

peterzimon commented on ghost/i18n/locales/hu/comments.json:
```diff
@@ -3,68 +3,68 @@
     "Sent": "Elküldve",
     "Show": "Mutat",
-    "Show {{amount}} more replies": "Mutass még {{amount}} választ",
+    "Show {{amount}} more replies": "Mutass {{amount}} még válaszokat",
```
Hey @eszterik!!
Amazing work, thank you so much, it really sounds much better in Hungarian. The only one I found that I'd go back to the original translation is this one: `{{amount}}` is a number and so e.g. for 5 more replies the original is "Mutass még 5 választ" which sounds better than "Mutass 5 még válaszokat"

Otherwise all looks great, thanks again!
(I'm also from Hungary :) )

eszterik replied:
Hello @peterzimon!
Thank you for checking it! I'm glad that another Hungarian speaker thinks it works. :) I’ll change the sentence you mentioned - you’re right, this sounds better. :) Best, Eszter

eszterik replied:
Hello @cathysarisky,
I corrected what Peter mentioned. Thank you for reaching out to him!
And thank you again for your patience and careful work—I know I was sometimes (or maybe more than sometimes!:) a bit clueless. But at least I finally learned more or less how GitHub works! :) 
Best,
Eszter

peterzimon replied:
Hi @eszterik,
Can you take care of resolving the conflicts in the json files so we're able to merge your changes?
Thank you

cathysarisky replied:
Hi @eszterik - I'm guessing you're going to need some directions on how to do that ^^ ? 

Here are directions!  
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github

eszterik replied:
Hello @cathysarisky, I'm sorry, but it is still not clear where and how should I do this things...Could you help me? Thanks!

cathysarisky replied:
I think @9larsons said he'd take care of it. :)

eszterik replied:
Great, thank you for your reply @cathysarisky. Does this mean I don't have anything more to do?

eszterik committed:
Added corrected Hungarian locale for Portal

9larsons committed:
Merge branch 'main' into main

9larsons committed:
Merge branch 'main' into main

9larsons committed:
Merge branch 'main' into main

9larsons committed:
Updated i18n files

agota1229 commented:
Hi!

Do you have any idea when these changes can be seen? I tried to refresh, but all I see is still the old Hungarian version

cathysarisky commented:
The changes are live in Portal 2.50, with Ghost 5.113.  If you're self hosting, please update Ghost.  If you're on Ghost Pro and aren't seeing the changes, you may need to wait one more day.  Typically Ghost Pro lags the release by a couple of business days, so I'd expect most Ghost Pro sites to get the update by today.
